pkgs/by-name/te/tenv/package.nix

```
@@ -0,0 +1,43 @@
+{ buildGoModule, fetchFromGitHub, lib, testers, tenv }:
```

```suggestion
{ buildGoModule, fetchFromGitHub, lib, tenv, testers }:
```
I'm in favor of alphabetical order. But there is no consensus on this.
So, now I'm trolling. =]

(I'm waiting build to test version + cross build, etc.)